telegram/replymarkup.py

```diff
@@ -31,3 +35,13 @@ class ReplyMarkup(TelegramObject):
     """
 
     __slots__ = ()
+
+    @staticmethod
+    def _check_keyboard_type(
+        keyboard: Sequence[Sequence[Union['KeyboardButton', 'str', 'InlineKeyboardButton']]]
+    ) -> bool:
```

Comment:
shouldn't this rather just accept `object`? E.g. a confused user might just pass an `InlineKeyboardButton`, not even a simple list

Reply:
I guess so, I just 'combined' the type hints of both the keyboards since its intended to be used only for them

Reply:
Okay nvm, mypy gives the error: 
`telegram/replymarkup.py:42: error: "object" has no attribute "__iter__"; maybe "__str__" or "__dir__"? (not iterable)  [attr-defined]`
 if you change it to `object`

Reply:
> Also it's not really meant to be used by users since its private.

true. by "pass" I meant pass to `Inline/ReplyKeyboardMarkup`. My point is that if the helper function only accepts the correct type, then it doesn't really test much ;)

Reply:
> Okay nvm, mypy gives the error:
> `telegram/replymarkup.py:42: error: "object" has no attribute "__iter__"; maybe "__str__" or "__dir__"? (not iterable) [attr-defined]`
> if you change it to `object`

that's exactly why it should be `object` :)

Reply:
oh I see, i'll just add a ignore comment then